backend/src/Controller/Admin/StoreOwnerSubscription/AdminStoreSubscriptionController.php

```diff
@@ -99,4 +113,183 @@ public function deleteAllStoreFutureSubscriptionsByStoreOwnerId(int $storeOwnerI
 
         return $this->response($result, self::DELETE);
     }
+
+    /**
+     * admin: create subscription
+     * @Route("createsubscription", name="createSubscriptionByAdmin", methods={"POST"})
+     * @IsGranted("ROLE_ADMIN")
+     * @param Request $request
+     * @return JsonResponse
+     *
+     * @OA\Tag(name="Subscription")
+     *
+     * @OA\Parameter(
+     *      name="token",
+     *      in="header",
+     *      description="token to be passed as a header",
+     *      required=true
+     * )
+     *
+     * @OA\RequestBody(
+     *      description="create subscription",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="string", property="status_code"),
+     *          @OA\Property(type="string", property="msg"),
+     *          @OA\Property(type="object", property="Data",
```

Comment:
request body has no Data, msg, and status code. :)

Reply:
this is new style ^_^.
done